pages/ar/developing/developer-faqs.mdx

```diff
@@ -110,7 +110,7 @@ curl -X POST -d '{ "query": "{indexingStatusForCurrentVersion(subgraphName: \"or
 - Ethereum mainnet
 - Kovan
 - Rinkeby
-- Ropsten
+- Goerli
```

Comment:
Goerli was already in that list (just below).

Reply:
Also, there is an instance of "Kovan" at the bottom of this list.

Reply:
Good catch, hadn't saved that file